projects/pedal/inc/throttle.h

```diff
@@ -0,0 +1,58 @@
+#pragma once 
+
+// Module for the throttle 
+
+#include <stdint.h>
+
+#include "ads1015.h"
+#include "exported_enums.h"
+#include "soft_timer.h"
+#include "interrupt.h"
+#include "status.h"
+
+
+// Time period between every throttle pedal reading 
+#define THROTTLE_UPDATE_PERIOD_MS 10 
+
+
+// Throttle Channels: 
+// Two channels are used for redundancy in case throttle reading from the primary channel goes bad. 
+/*typedef enum {
```

Comment:
we don't allow /* */ comments, if you run `make format` and `make lint` it'll either fix it for you, or tell you to change it.